src/main/java/com/pinterest/secor/uploader/S3UploadManager.java

```diff
@@ -69,7 +71,11 @@
     private static final String S3 = "S3";
     private static final String CUSTOMER = "customer";
 
+    private final String S3_PATH;
```

Comment:
This instance variable should not be uppercased.  The upper case are for constants.